artifact/keyfactor/signer.go

```diff
@@ -0,0 +1,213 @@
+/*
+Copyright 2023 The Keyfactor Command Authors.
```

Comment:
Note to self: add these files as an exception for the `test:check-license-source` CI job